.github/workflows/deploy-enterprise.yml

```diff
@@ -72,6 +72,27 @@ jobs:
             ami_id: /aws/service/ecs/optimized-ami/amazon-linux-2/recommended/image_id
             distribution_url: ''
 
+          - environment: hyp3-test-enterprise
+            domain: hyp3-test-enterprise.asf.alaska.edu
+            template_bucket: cf-templates-aubvn3i9olmk-us-west-2
+            image_tag: test
+            product_lifetime_in_days: 14
+            quota: 0
+            deploy_ref: refs/heads/develop
```

Comment:
Since this workflow is only triggered
```yaml
on:
  push:
    branches:
      - main
```
you'll need to:
- [ ]  add `develop` to the triggers
- [ ] add a `deploy_ref` to every matrix element / environment
- [ ] add the deploy ref condition:
       https://github.com/ASFHyP3/hyp3/blob/develop/.github/workflows/deploy-daac.yml#L74

Reply:
Actually, thinking more about this, I think we might want to put this in a separate `deploy-enterprise-test.yml` workflow -- the way we have it setup with the deploy ref, it'll "try" to deploy to every environment but quickly succeed by skipping the deploy job if the ref != to the one specified.

That'd mean, we'd be asked for approval to deploy the prod enterprise stacks on any merge to main _or_ develop, which is rather undesirable. 

Putting it in a separate workflow file only triggered on `develop` (and not requiring reviews in the action env) I think is the better option.

Reply:
Another thought, we could keep it in the same workflow and see how this env setting plays out:
![image](https://user-images.githubusercontent.com/7882693/176732598-3cf6953a-0070-46de-a22e-f60ba99e78d3.png)

I *think* that'd just make the deploy jobs fail if not on the deployment branch, but it's possible it skips instead...

Reply:
@jhkennedy How do I implement:
> not requiring reviews in the action env